From f0c3d677ebe124ff8b168253d3d69879a5cc023d Mon Sep 17 00:00:00 2001
From: Aaron Twining Myers <atm@apache.org>
Date: Wed, 8 Aug 2012 17:21:27 +0000
Subject: [PATCH 0379/1357] HDFS-3773. TestNNWithQJM fails after HDFS-3741. Contributed by Aaron T. Myers.

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/HDFS-3077@1370846 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 58ce6213466db4777a2e48a3655352491c5e39f1)
(cherry picked from commit d6ca63104ee98851db9d44e42e5ba416363f6ad2)
---
 .../hadoop-hdfs/CHANGES.HDFS-3077.txt              |    2 ++
 .../hdfs/qjournal/client/QuorumJournalManager.java |    2 +-
 2 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/CHANGES.HDFS-3077.txt b/hadoop-hdfs-project/hadoop-hdfs/CHANGES.HDFS-3077.txt
index 6ec8a4c..ca65f2b 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/CHANGES.HDFS-3077.txt
+++ b/hadoop-hdfs-project/hadoop-hdfs/CHANGES.HDFS-3077.txt
@@ -12,3 +12,5 @@ HDFS-3693. JNStorage should read its storage info even before a writer becomes a
 HDFS-3725. Fix QJM startup when individual JNs have gaps (todd)
 
 HDFS-3741. Exhaustive failure injection test for skipped RPCs (todd)
+
+HDFS-3773. TestNNWithQJM fails after HDFS-3741. (atm)
diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/QuorumJournalManager.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/QuorumJournalManager.java
index 786e9f5..7846e59 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/QuorumJournalManager.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/QuorumJournalManager.java
@@ -76,7 +76,7 @@ public class QuorumJournalManager implements JournalManager {
   
   private final AsyncLoggerSet loggers;
   
-  QuorumJournalManager(Configuration conf,
+  public QuorumJournalManager(Configuration conf,
       URI uri, NamespaceInfo nsInfo) throws IOException {
     this(conf, uri, nsInfo, IPCLoggerChannel.FACTORY);
   }
-- 
1.7.0.4

